METAPROGRAMMING.md

```diff
@@ -189,11 +191,10 @@ function bound(elementDescriptor) {
   // Return both the original method and a bound function field that calls the method.
```

Comment:
(A few lines above, GitHub doesn't let me to comment there).
The destructuring needs to be updated to remove `descriptor`.